server/member/member.go

```diff
@@ -76,7 +77,16 @@ func (m *Member) GetLeader(ctx context.Context) (*GetLeaderResp, error) {
 		return nil, ErrInvalidLeaderValue.WithCause(err)
 	}
 
-	return &GetLeaderResp{Leader: leader, Revision: leaderKv.ModRevision, IsLocal: leader.GetId() == m.ID}, nil
+	return &getLeaderResp{Leader: leader, Revision: leaderKv.ModRevision, IsLocal: leader.GetId() == m.ID}, nil
+}
+
+// GetLeader gets the leader of the cluster with memory cache.
+// return error if no leader found.
+func (m *Member) GetLeader(_ context.Context) (*metastoragepb.Member, bool, error) {
```

Comment:
How about renaming this method to `GetLeaderAddr`, and returns a struct called LeaderAddr with only two fields:
- address of the leader
- boolean `Local`